examples/ct/handlers.go

```diff
@@ -182,22 +186,44 @@ func NewLogContext(logID int64, prefix string, trustedRoots *PEMCertPool, rpcCli
 	return ctx
 }
 
+// Handlers returns a map from URL paths (with the given prefix) and AppHandler instances
+// to handle those entrypoints.
+func (c LogContext) Handlers(prefix string) PathHandlers {
```

Comment:
Add a test for Handlers?

I don't think we need to check the entirety of the map (or it'll begin to look like a change detector test), but testing the prefix logic and checking that all methods from Entrypoints are represented seems reasonable.